pep-0639.rst

```diff
@@ -2292,8 +2121,145 @@ Some additional examples of valid ``License-Expression`` values::
     License-Expression: LicenseRef-Proprietary
 
 
-Appendix 2. License Documentation in Python
-===========================================
+Appendix: User Scenarios
+========================
+
+The following covers the range of common use cases from a user perspective,
+providing straightforward guidance for each. Do note that the following
+should **not** be considered legal advice, and readers should consult a
+licensed attorney if they are unsure about the specifics for their situation.
```

Comment:
Is "licensed attorney" the best phrasing for world-wide meaning? Perhaps "licenced legal practitioner", although such a turn of phrase will never have the exactness of a specific jurisdiction's nomenclature.

Reply:
Good point; that's the standard legal disclaimer phraseology in the US but may not be as intelligible worldwide. I'll revise it to your suggestion.